env/manifests/kafka_micro.yaml

```diff
@@ -16,7 +16,7 @@ spec:
   selector:
     app: kafka
 ---
-apiVersion: policy/v1beta1
+apiVersion: policy/v1
```

Comment:
I had some issues with `v1` in the version of minikube that was mentioned in the book. I'd rather keep it, if possible.